src/storage/loot.ts

```diff
@@ -110,6 +111,13 @@ export async function createLoot(
             .returningAll()
             .executeTakeFirstOrThrow();
 
+        for (const attributeId of template.initialAttributes || []) {
```

Comment:
```suggestion
        for (const attributeId of template.initialAttributes ?? []) {
```

Reply:
man hätte auch `if (template.initialAttributes) {` davor machen können. Fänd ich noch besser, aber so is auch gut